prisma/schema.prisma

```diff
@@ -18,4 +17,121 @@ model User {
   email     String   @unique
   name      String?
   createdAt DateTime @default(now())
+}
+// ============================================
+// MÓDULO TESTS
+// ============================================
+
+enum QuestionType {
+  MCQ
+  PROGRAMMING
+}
+
+enum DifficultyLevel {
+  EASY
+  MEDIUM
+  HARD
+}
+
+model Company {
+  id        String   @id @default(uuid())
+  name      String
+  logo      String?
+  tests     Test[]
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+
+  @@map("companies")
+}
+
+model Test {
+  id          String          @id @default(uuid())
+  name        String
+  description String
+  company     Company         @relation(fields: [companyId], references: [id])
+  companyId   String
+  difficulty  DifficultyLevel
+  duration    Int
+  isActive    Boolean         @default(true)
+  questions   Question[]
+  sessions    TestSession[]
+  submissions Submission[]
+  createdAt   DateTime        @default(now())
+  updatedAt   DateTime        @updatedAt
+
+  @@index([companyId])
+  @@index([difficulty])
+  @@index([isActive])
+  @@map("tests")
+}
+
+model Question {
+  id              String          @id @default(uuid())
+  test            Test            @relation(fields: [testId], references: [id], onDelete: Cascade)
+  testId          String
+  type            QuestionType
+  text            String
+  difficulty      DifficultyLevel
+  points          Int
+  timeLimit       Int?
+  options         Json?
+  programmingData Json?
+  order           Int
+  createdAt       DateTime        @default(now())
+  updatedAt       DateTime        @updatedAt
+
+  @@index([testId])
+  @@index([type])
+  @@map("questions")
+}
```

Comment:
TestSession has userId: String but no relation defined to the User model. Without an explicit relation Prisma and DB-level constraints cannot guarantee the user exists.

**Sugerencia:**
```typescript
If your app has a User model, add a relation line: e.g.
  user   User   @relation(fields: [userId], references: [id])
  userId String
This enables Prisma-level joins and enforces integrity. If you intentionally keep it denormalized, document the reason. Priority: important
```